core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -34,12 +35,30 @@ oppia.directive('simpleEditorSidebar', [
           /* This initializes Perfect Scrollbar on the simple editor sidebar.
            Perfect scrollbar is needed to show scrollbar on all major browsers.
            */
-          Ps.initialize(document.querySelector('simple-editor-sidebar'));
+          var container = document.querySelector('simple-editor-sidebar');
+          Ps.initialize(container);
           $scope.SUBFIELD_LABELS = [
             'Multiple choice', 'Correct answer', 'Hints', 'Bridge text'];
           $scope.questionList = SimpleEditorManagerService.getQuestionList();
           $scope.ID_PREFIX = QuestionIdService.SIDEBAR_PREFIX;
           $scope.sidebarModeService = SimpleEditorSidebarModeService;
+
+          $scope.canAddNewQuestion = (
+            SimpleEditorManagerService.canAddNewQuestion);
+
+          $scope.addNewQuestion = function() {
+            if (!SimpleEditorManagerService.canAddNewQuestion()) {
+              return;
+            }
+            $scope.sidebarModeService.setModeToReadonly();
+            SimpleEditorManagerService.addNewQuestion();
+            $timeout(function() {
+              $scope.scrollToQuestion($scope.questionList.getLastQuestion());
+              container.scrollTop = container.scrollHeight;
```

Comment:
The new question is not getting added in the sidebar by the call of line 57. @seanlip Do you have any insight on why this might be happening?